try-greg.py

```diff
@@ -0,0 +1,59 @@
+import sys
+import math
+
+def make_blanket( blanket_radius, blanket_width, blanket_height, shell_thickness ):        
+    cubit.cmd( f"create brick x {blanket_height} y {blanket_width} z {blanket_width}" )
+    shell_volume_id = cubit.get_last_id( "volume" )
+    cubit.cmd( f"modify curve {get_closest_curve_in_volume_to_location( shell_volume_id, (blanket_height/2, blanket_width/2, 0) )} blend radius {blanket_radius}" )
+    cubit.cmd( f"modify curve {get_closest_curve_in_volume_to_location( shell_volume_id, (-blanket_height/2, blanket_width/2, 0) )} blend radius {blanket_radius}" )
+
+    cubit.cmd( f"create sheet offset from surface in vol {shell_volume_id} offset {-1*shell_thickness} " )
+    inner_volume_id = cubit.get_last_id( "volume" )
+
+    before_vol_ids = cubit.get_entities( "volume" )
+    cubit.cmd( f"subtract volume {inner_volume_id} from volume {shell_volume_id} keep_tool" )
+    after_vol_ids = cubit.get_entities( "volume" )
+    inner_volume_id, shell_volume_id = get_operation_return_ids( before_vol_ids, after_vol_ids )
+    
+    cubit.cmd( f"block 1 volume {cubit.get_id_string( shell_volume_id )}" )
+    cubit.cmd( f"block 2 volume {cubit.get_id_string( inner_volume_id )}" )
+    cubit.cmd( "block 1 name 'OuterShell'" )
+    cubit.cmd( "block 2 name 'InnerVolume'" )
```

Comment:
Just an example of the `get_id_string()` method.  Not much use here, since both `shell_volume_id` and `inner_volume_id` are scalars, but in the case that they happened to be a list this method will convert the list into a Cubit-parseable id string.  For example:

```python
%> cubit.get_id_string( [1, 2, 3, 4] )
'1 to 4'
```